db/seeds.rb

```diff
@@ -5,3 +5,32 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+[
+'Scotch Ale',
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.